src/renderer/components/screens/MainScreen.tsx

```diff
@@ -240,54 +241,55 @@ export default function MainScreen() {
               </NavbarHeading>
             )}
             {selectedChat.chat && (
-              <span className='views'>
```

Comment:
Why are these changes needed?

Reply:
to hide the chat context menu when no chat is selected, basically to fix a bug